### ISSUE TITLE
Looking to build this to support cel_redstone-powerpc

### ISSUE THREAD

**Issue by mguzsklk** [opencomputeproject/OpenNetworkLinux#954]
Hi Folks,  I have a Celestica D2020 which is cel_redstone-powerpc.   I used the powerpc version of ONL but realized that there is no onlp for this chipset.  I understand that this is probably a closed source or third party component that is not by default included in your builds. 

I've been scouring the web and have thus far failed to find any NOS that is freely installable on this device due to its powerpc-ness.  The brocade switch chip is not a problem but none of the NOSes out there support the powerpc cpu.   The thing came with onie and Fastpath but due to a PBKaC it got erased.  Now I'm stumped.  

Long story short.  Is there any know way to get ONL working for this switch Model?

Kind Regards
Mark Guz
